Legacy/BootPlatform/GraphicsConsoleDxe/GraphicsConsole.c

```diff
@@ -1153,6 +1153,9 @@ GraphicsConsoleConOutSetMode (
                           ModeData->GopHeight,
                           0
                           );
+      if (EFI_ERROR (Status)) {
```

Comment:
Here and below no Status checks are intentional as Blt may fail before mode-set. Should probably write a comment to avoid a future miscorrection.